zhcsshim.go

```diff
@@ -48,6 +45,28 @@ var (
 	procTerminateComputeSystem                     = modvmcompute.NewProc("TerminateComputeSystem")
```

Comment:
I don't agree. We need to support the old APIs for a little while in case we need to revendor hcsshim to some code that hasn't been updated yet. Groups other than docker have taken a dependency on hcsshim.